src/popup/components/AdvancedTransactionDetails.vue

```diff
@@ -0,0 +1,130 @@
+<template>
+  <DetailsItem
+    expandable
+    class="advanced-transaction-details"
+    :label="$t('transaction.advancedDetails')"
+  >
+    <DetailsItem
+      v-if="callData"
+      :label="$t('transaction.data')"
+    >
+      <Tabs class="tabs">
+        <Tab
+          v-for="({ name, text }) in dataTabs"
+          :key="name"
+          :data-cy="name"
+          :name="name"
+          :text="text"
+          :active="activeTab === name"
+          @click="setActiveTab(name)"
+        />
+      </Tabs>
+      <div class="tabs-content">
+        <!-- Decoded Data -->
+        <DetailsItem v-if="activeTab === dataTabs[0].name">
+          <Panel v-if="decodedCallData?.args">
+            <PanelTableItem
+              v-for="([key, value]) in Object
+                .entries(decodedCallData?.args as any) || []"
+              :key="key"
+              :name="key"
+            >
+              <div class="scrollable">
+                {{ value }}
+              </div>
+            </PanelTableItem>
+          </Panel>
+          <InfoBox
+            v-else
+            type="warning"
+            :text="$t('transaction.decodingDataFailed')"
+          />
+        </DetailsItem>
+        <!-- Raw Data -->
```

Comment:
I suggest to add empty line between the different tab contents (before this line).